src/python_pybind11/src/Helpers.cc

```diff
@@ -164,23 +164,23 @@ void defineMathHelpers(py::module &m)
    .def("parse_float",
         &gz::math::parseFloat,
         "parse string into an float")
-   .def("ign_sphere_volume",
+   .def("gz_sphere_volume",
```

Comment:
There are not being tick-tocked, right? Can we have both the old and new versions? Not sure how trivial it would be to add warnings.

Reply:
This should do it, I think!